lib/datadog/grok/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "datadog-grok"
+version = "0.1.0"
+authors = ["Vector Contributors <vector@timber.io>"]
+edition = "2018"
+build = "src/build.rs" # LALRPOP preprocessing
+
+[dependencies]
+serde = { version = "1.0.126", default-features = false, features = ["derive"] }
+serde_yaml = "0.8"
+lalrpop-util = "0.19"
+thiserror = "1"
+lazy_static = "1.3.0"
+regex = "1"
+grok = "1"
+indoc = { version = "1.0.3", default-features = false }
+inventory = { version = "0.1.10", default-features = false }
+async-trait = "0.1.50"
+typetag = { version = "0.1.7", default-features = false }
+derivative = { version = "2.2.0", default-features = false }
+itertools = { version = "0.10.1", default-features = false }
+serde_json = { version = "1.0.64", default-features = false }
+percent-encoding = { version = "2.1.0", default-features = false }
+bytes = "1.0.0"
+enum_derive = "0.1.3"
+custom_derive = "0.1.2"
+strum = "0.21.0"
+strum_macros = "0.21"
```

Comment:
I realize these may not have features, but I think we've been erring towards using `default-features = false` everywhere just-in-case that changes.

You can also use https://github.com/est31/cargo-udeps here to see if there are any unused dependencies.

Reply:
Thanks, that's good to know 👍